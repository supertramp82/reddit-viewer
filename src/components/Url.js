import React from 'react';

function Url({ url }) {
  return (
    <div className="url">
      <a href="#">{url}</a>
    </div>
  );
}

export default Url;
