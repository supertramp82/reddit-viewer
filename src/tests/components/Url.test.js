import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Url from '../../components/Url';

test('should render Url correctly', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Url url="http://www.test.com" />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
