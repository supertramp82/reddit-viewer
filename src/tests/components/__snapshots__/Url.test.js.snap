// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Url correctly 1`] = `
<div
  className="url"
>
  <a
    href="http://www.test.com"
  >
    http://www.test.com
  </a>
</div>
`;
